ur_robot_driver/CMakeLists.txt

```
@@ -156,6 +162,15 @@ if(BUILD_TESTING)
   find_package(ur_description REQUIRED)
   find_package(ur_msgs REQUIRED)
   find_package(launch_testing_ament_cmake)
-  add_launch_test(test/integration_test_1.py)
-  add_launch_test(test/integration_test_2.py)
+
+  if(${UR_ROBOT_DRIVER_BUILD_INTEGRATION_TESTS})
```

A comment for clarity is probably a good thing to do here.
```suggestion
  # The integration tests will most probably not run inside the buildfarm, since we need to be able to create new docker 
  # containers and provide the ursim_net for the container running the CI which we cannot influence
  if(${UR_ROBOT_DRIVER_BUILD_INTEGRATION_TESTS})
```